<ui version="4.0" >
 <class>browseDialogClass</class>
 <widget class="QDialog" name="browseDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Browse dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="acceptDrops" >
      <bool>true</bool>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Compounds</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="3" >
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tableWidget" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <property name="columnCount" >
            <number>4</number>
           </property>
           <column/>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="3" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="title" >
          <string>Compound information</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="leftMargin" >
             <number>0</number>
            </property>
            <property name="topMargin" >
             <number>0</number>
            </property>
            <property name="rightMargin" >
             <number>0</number>
            </property>
            <property name="bottomMargin" >
             <number>0</number>
            </property>
            <property name="horizontalSpacing" >
             <number>6</number>
            </property>
            <property name="verticalSpacing" >
             <number>6</number>
            </property>
            <item rowspan="2" row="1" column="0" >
             <widget class="QListWidget" name="listWidget" />
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Names</string>
              </property>
             </widget>
            </item>
            <item rowspan="2" row="1" column="1" >
             <widget class="QListWidget" name="listWidget_2" />
            </item>
            <item row="0" column="1" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Reactions</string>
              </property>
             </widget>
            </item>
            <item rowspan="2" row="1" column="2" >
             <widget class="QListWidget" name="listWidget_3" />
            </item>
            <item row="0" column="2" >
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>Transitions</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_3" >
           <property name="maximumSize" >
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>Show selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2" >
           <property name="maximumSize" >
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text" >
            <string>Show all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>111</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2" >
        <widget class="QPushButton" name="pushButton_4" >
         <property name="maximumSize" >
          <size>
           <width>141</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string>Open transition window</string>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Reactions</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>5</number>
         </property>
         <property name="topMargin" >
          <number>5</number>
         </property>
         <property name="rightMargin" >
          <number>5</number>
         </property>
         <property name="bottomMargin" >
          <number>5</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tableWidget_2" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <property name="columnCount" >
            <number>4</number>
           </property>
           <column/>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2" >
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QCheckBox" name="checkBox" >
           <property name="text" >
            <string>show selected in dataset window</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QCheckBox" name="checkBox_2" >
           <property name="text" >
            <string>display metabolite compartments</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2" >
           <property name="title" >
            <string>Information</string>
           </property>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QGridLayout" >
              <property name="leftMargin" >
               <number>0</number>
              </property>
              <property name="topMargin" >
               <number>0</number>
              </property>
              <property name="rightMargin" >
               <number>0</number>
              </property>
              <property name="bottomMargin" >
               <number>0</number>
              </property>
              <property name="horizontalSpacing" >
               <number>6</number>
              </property>
              <property name="verticalSpacing" >
               <number>6</number>
              </property>
              <item row="0" column="0" >
               <widget class="QLabel" name="label_3" >
                <property name="text" >
                 <string>Equation</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="QLineEdit" name="lineEdit" />
              </item>
              <item row="1" column="0" >
               <widget class="QLabel" name="label_4" >
                <property name="text" >
                 <string> Name</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="QLineEdit" name="lineEdit_2" />
              </item>
              <item row="2" column="1" >
               <widget class="QComboBox" name="comboBox" />
              </item>
              <item row="2" column="0" >
               <widget class="QLabel" name="label_5" >
                <property name="text" >
                 <string>RPair Tags</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1" >
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="maximumSize" >
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title" >
          <string>RPairs</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QListWidget" name="listWidget_4" >
            <property name="maximumSize" >
             <size>
              <width>1000</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="pushButton_6" >
           <property name="maximumSize" >
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="text" >
            <string>Show selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton" >
           <property name="maximumSize" >
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="text" >
            <string>Show all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
